public/app/plugins/datasource/influxdb/module.tsx

```diff
@@ -0,0 +1,17 @@
+import InfluxDatasource from './datasource';
+import { InfluxQueryCtrl } from './query_ctrl';
+import { InfluxLogsQueryField } from './components/InfluxLogsQueryField';
+import InfluxStartPage from './components/InfluxStartPage';
+import { DataSourcePlugin } from '@grafana/data';
+import ConfigEditor from './components/ConfigEditor';
+
+class InfluxAnnotationsQueryCtrl {
+  static templateUrl = 'partials/annotations.editor.html';
+}
+
```

Comment:
Why does this need to change to a `.tsx` file?

Reply:
That's what they did the grafana test db which was the only one in React- followed suit.

Reply:
tsx is required if you include jsx in the file -- otherwise, .ts is fine.

In general for a PR we should avoid changing the name unless there is a reason to change it (the bar can be low)